app/main/forms/suppliers.py

```diff
@@ -82,3 +82,36 @@ class EmailAddressForm(Form):
         DataRequired(message="You must provide an email address."),
         EmailValidator(message="You must provide a valid email address."),
     ])
+
+
+class CompanyOrganisationSizeForm(Form):
+    OPTIONS = [
+        {
+            "value": "micro",
+            "label": "Micro",
+            "description": "Under 10 employees and 2 million euros or less in either annual turnover or balance "
+                           "sheet total",
+        },
+        {
+            "value": "small",
+            "label": "Small",
+            "description": "Under 50 employees and 10 million euros or less in either annual turnover or balance "
+                           "sheet total",
+        },
+        {
+            "value": "medium",
+            "label": "Medium",
+            "description": "Under 250 employees and either 50 million euros or less in either annual turnover or "
+                           "43 million euros or less in annual balance sheet total",
+        },
+        {
+            "value": "large",
+            "label": "Large",
+            "description": "250 or more employees and either over 50 million euros in annual turnover or over 43 "
+                           "million euros in balance sheet total",
+        },
+    ]
+
+    organisation_size = RadioField('Organisation size',
+                                   validators=[InputRequired(message="You must choose an organisation size.")],
```

Comment:
What's the difference between `DataRequired` (which we already import) and `InputRequired`?

Reply:
Essentially, `InputRequired` allows us to specify a custom message and `DataRequired` doesn't. WTForms also recommends using InputRequired generally.

After digging into what's actually going on under the hood this morning, it looks like DataRequired has some kind of incompatibility with SelectField. InputRequired makes sure that the user has submitted some kind of value; DataRequired makes sure the user has submitted a value that evaluates to 'truthy' after being coerced to whatever format the form wants.

There's an existing issue on the project that I've tacked my little note onto: https://github.com/wtforms/wtforms/issues/289

Reply:
That's strange. We have a custom message on a `DataRequired` validator on the Briefs FE: https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/master/app/main/forms/awards.py#L11. However if `InputRequired` is recommended then by all means we should use that instead (we use both on the Briefs FE!).

Reply:
I suspect that works because you're coercing to an int rather than a string. int(None) throws an error, which I guess gets caught and handled. str(None) ends up with 'None' that then fails  a later check. Sounds pretty hinky all-in-all 🤔 

But yeah, it sounds like InputRequired is what we should be using instead.

Reply:
@Wynndow do you think we should maybe change our validators on address and country forms?

Reply:
Yep, it's probably a good idea.